content/blog/2016/2016-06-11-gsoc-jenkins-support-core-plugin-improvements.adoc

```diff
@@ -0,0 +1,82 @@
+---
+layout: post
+title: "GSoC Project Intro: Support Core Plugin Improvements"
+tags:
+- core
+- gsoc
+- plugin
+- support-core
+author: minudika
+---
+
+=== About me
+
+I am Minudika Malshan, an undergraduate student in Computer Science and Engineering from University of Moratuwa, Sri Lanka.In the field of Computer Science.
+
+As a person who is passionate in open source software development and seeking for new knowledge and experience , I am willing to give my contribution for this project.
+
+https://www.linkedin.com/in/minudika[LinkedIn] | https://twitter.com/minudika[Twitter]
+
+=== Abstract
+
+The Support-Core Plugin provides the basic infrastructure for generating "bundles" of support information with Jenkins.
+There are two kinds of bundles.
+
+* Automatic bundles : bundles which are get saved in $JENKINS_HOME/support once per hour starting 15 seconds after Jenkins starts the plugin.
```

Comment:
* NIT: generally, in English, colons (:) have no spaces before. But they do in french, so in the end that might be great as-is ;-)
* "bundles which are get saved". Missing words? you mean something like "bundles which get *generated* are saved in ..."?